tsconfig.json

```diff
@@ -1,24 +1,14 @@
 {
```

Comment:
I've simplified the TypeScript configuration a bit by putting only the essential configuration here. Configurations for specific tasks such as the build, linting and testing all live in their own configs that extend from the root config.